src/graphql/typeDefs/dataTypes/enums/status.graphql

```diff
@@ -6,4 +6,5 @@ enum Status {
   WITHDREW
   WAIT_LIST
   CANCELLED
+  SCHEDULED
```

Comment:
Good thought. Do we have a flag, or condition for sessions which cannot be rescheduled?

Reply:
good thought.. nope..  I was just starting to think about things like your above statement and how we might want to work this a bit differently to support a richer set of scenarios in the future.